src/pages/Dashboards/Supplier/SupplierList/components/AddEditSupplierModal.tsx

```
@@ -0,0 +1,286 @@
+const AddEditSupplierModal: React.FC<AddEditSupplierModalProps> = ({
+  show,
+  onHide,
+  isEdit,
+  eventData,
+  onSubmit,
+}) => {
+  const validation: any = useFormik({
+    enableReinitialize: true,
+
+    initialValues: {
+      name: (eventData && eventData.name) || "",
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)